public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -164,41 +190,53 @@ export class PanelInspector extends PureComponent<Props, State> {
   }
 
   renderDataTab() {
-    const { data, selected } = this.state;
+    const { data, selected, transformation } = this.state;
     const styles = getStyles();
 
     if (!data || !data.length) {
       return <div>No Data</div>;
     }
 
-    const choices = data.map((frame, index) => {
+    const dataFrames = data.map((frame, index) => {
       return {
         value: index,
         label: `${frame.name} (${index})`,
       };
     });
 
-    const processed = applyFieldOverrides({
-      data,
-      theme: config.theme,
-      fieldConfig: { defaults: {}, overrides: [] },
-      replaceVariables: (value: string) => {
-        return value;
-      },
-    });
+    const processed = transformDataFrame(
+      [{ id: transformation, options: { byField: 'Time' } }],
```

Comment:
these options will only work for the `seriesToColumns` transformer, right? If the plan is to add more alternatives here it might be a good idea to keep the options together with the selected transform so they can be applied together.

Reply:
I prefer to cross that bridge when we get to it.